pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_config_repo.adoc

```diff
@@ -1,12 +1,14 @@
 [id="configuring-a-repository-as-a-content-source-for-pantheon_{context}"]
 
-= Configuring a repository as a content source for Pantheon
+= Configuring a repository as a content source for {ProductShortName}
+
+In order to enable a repository as a content source for {ProductShortName}, you need to configure the repository by adding a configuration file to the root directory.
 
 .Procedure
 
-. In a text editor, paste the content of the sample configuration file in xref:pantheon-yaml-file_{context}[] into a new text file. Name the file [filename]`pantheon2.yml`. This file is the Pantheon configuration file for your repository.
+. In a text editor, paste the content of the sample configuration file in xref:pantheon-yaml-file_{context}[] into a new text file. Name the file [filename]`pantheon2.yml`. This file is the {ProductShortName} configuration file for your repository.
```

Comment:
Same comment here, `pantheon -S > pantheon2.yml`

Reply:
@benradey As you mentioned, you need to install the uploader tool locally to be able to use this command, so I didn't document it that way.